phpunit.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <phpunit bootstrap="../../tests/bootstrap.php"
 		 verbose="true"
+	 	 failOnWarning="true" strict="true"
```

Comment:
I know, cosmetic, but all other phpunit.xml files have if strict is present the strict directive in a new line...